examples/libs/libpng/index.rst

```diff
@@ -1,4 +1,4 @@
-Open a PNG image from file
+Open a PNG image from file and variable
```

Comment:
The decoration under the title should have the same amount of characters as the title.
```rst
Open a PNG image from file and variable
---------------------------------------
```

Reply:
Done